source/includes/_feature-versioning.md

```diff
@@ -75,12 +75,30 @@ HTTP/1.1 204 No Content
 *or*
 
 HTTP/1.1 202 Accepted
-Location: https://beta.crunch.io/api/progress/{progress_id}/
 ```
 
-The POST to the actions catalog tells the original dataset to replay a set of actions; since we specify a "dataset" url, we are telling it to replay all actions from the forked dataset. Crunch keeps track of which actions are already common between the two datasets, and won't try to replay those. You can even make further changes to the forked dataset and merge again and again.
+```json
+{
+    "element": "shoji:view",
+    "self": "/api/datasets/{id}/actions/",
```

Comment:
Not necessary, but the Shoji controllers will add it to all shoji responses. Not sure if it's worth the effort to work around it?

Although, GETting to that self url will not give you the same response.